filters/node.js

```diff
@@ -0,0 +1,29 @@
+import flatten from 'flat'; 
+
+export class NodeFilter {
+
+    async parseResource( resource ) {
+        const segments = [];
+        for(const [key, value] of Object.entries(flatten(resource))){
+            if (key.indexOf("@")===-1) {
+                const seg = {
+                    sid: key,
+                    str: value,
+                };
+                if (key.endsWith("_one")||key.endsWith("_other")) {
+                    seg.isSuffixPluralized = true;
+                }
+                segments.push(seg);
+            } else {
+                if(key.endsWith(".description")) {
+                    const entry = segments.find(e=>e.sid===key.replace("@","").replace(".description",""));
```

Comment:
this assumes that the annotation is always "after" the main property, which may not be the case.

Reply:
Fair.  Fixed.